util/src/test/java/io/grpc/util/SpiffeIdParserImplTest.java

```diff
@@ -0,0 +1,134 @@
+package io.grpc.util;
```

Comment:
In the reference example, we have a much larger corpus of test examples. Can we make sure all of those get ported to these tests?